libbpf-rs/src/map.rs

```diff
@@ -61,6 +61,20 @@ impl OpenMap {
         unsafe { libbpf_sys::bpf_map__set_inner_map_fd(self.ptr, inner.fd()) };
     }
 
+    /**
+     * Reuse a map created within the same userspace process.
```

Comment:
it doesn't have to be created within the userspace process, why these unnecessary and misleading details? Just something like "reuse provided BPF map FD instead of creating a new map" or something along those line would be more accurate.

You can get BPF map FD in many ways even if map wasn't created by your own process (e.g., look up by map ID, open from BPF FS, or even transfer FD over unix domain socket).

Reply:
I was trying to define the difference between `reuse_fd` and `reuse_pinned_map` in the comments to justify why I split some code from `reuse_pinned_map` into a new function.  What do you think would be the best way to do that, or is that not needed at all?

Reply:
I think the name is pretty clear, one reuses by map FD, another reuses map pinned in BPF FS. If someone doesn't know about pinning, they'll have to learn concepts before being able to use this API, I don't think there is a short cut.

Reply:
Sounds good. I'll remove that doc comment then.